README.md

```diff
@@ -66,6 +68,31 @@
 | Internet Explorer           | 10 & 11 |
 | Safari & Mobile Safari      | Current |
 
+## Internationalization(I18n)
```

Comment:
Can we pull this information out into terra-i18n/docs so we can pull it into our doc site?